### DISCUSSION TITLE
What is the lowest layer of Earth's atmosphere?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#136] [Galaxy Trivia] [answered]
**Hint:** 6-20 km

**Comment by Tynab** [comment, marked as answer]
The lowest layer of Earth's atmosphere is called the troposphere. It is the layer closest to the Earth's surface, extending from the surface up to an altitude of about 7 to 20 kilometers (4 to 12 miles) depending on the location on Earth. In the troposphere, air temperature generally decreases with increasing altitude. This layer contains almost all of the Earth's weather, including clouds, precipitation, and storms. It is also where most of the Earth's atmospheric gases, such as oxygen, nitrogen, and carbon dioxide, are found. The boundary between the troposphere and the layer above it, called the stratosphere, is known as the tropopause.
